### PR TITLE
Added changes to linux docs to show new process

### PR DESCRIPTION
**Pull request by enclave-tom**

### CODE REVIEW

File: docs/getting-started/installation.md [hunk 1]
```diff
@@ -70,93 +70,142 @@ You will need to install Enclave on at least two separate devices, systems or co
 
     Select your distribution:
 
-    === "Ubuntu / Debian"
+      === "Ubuntu / Debian"
```
Comment (enclave-marc-barry) [inline docs/getting-started/installation.md:73]: Hey Tom, this isn't quite right. The indentation is one too many one the whole file. Suggest running in the root of this repo, i'll boot up a local docker container that you can browse to at http://localhost:8000/getting-started/installation/ to preview the changes

```
PS C:\git\enclave\docs> .\run.ps1
```

**Review by enclave-marc-barry — CHANGES_REQUESTED** [review]

File: docs/getting-started/installation.md (hunk 1 above)
Comment (enclave-tom) [reply docs/getting-started/installation.md:73]: I've fixed the indentation but i may have found a bug in the code 
image

image
i now can't get back to the ubuntu instructions

**Comment by enclave-marc-barry** [comment]
I just pushed a tab fix on `installation.md`, can apply the same fix to linux.md tomorrow if you don't beat me to it

**Comment by enclave-tom** [comment]
> I just pushed a tab fix on `installation.md`, can apply the same fix to linux.md tomorrow if you don't beat me to it

My only suggestion would be to remove "amazon linux" as that also uses the same commands as in "other"

**Review by enclave-marc-barry — DISMISSED** [review]
Seems the tabs aren't a a future proof UI construct, so we'll no doubt need to re-visit again but this looks much better now, thanks Tom.

File: docs/getting-started/installation.md [hunk 2]
```diff
@@ -52,111 +52,153 @@ You will need to install Enclave on at least two separate devices, systems or co
...
+                    sudo enclave enrol
 
-        1. Install Enclave using our quick-start script
+            4. Provide your `Quick Start Key` to complete the installation
 
-                bash <(curl -Ss https://install.enclave.io/setup.sh)
+        === "Fedora"
 
-        2. Provide your `Quick Start Key` to complete the installation
+                <small>Requires at least Fedora 32.</small>
```
Comment (enclave-alistair) [inline docs/getting-started/installation.md:139]: Incorrect indentation here is causing all this to be a code block.

image

**Review by enclave-alistair — CHANGES_REQUESTED** [review]
One indentation issue, and we're missing the linux uninstall instructions for the new distros on /setup/linux.md.

**Pull request merged**